webtau-http/src/main/java/com/twosigma/webtau/http/validation/HttpValidationHandlers.java

```diff
@@ -23,6 +23,15 @@
 public class HttpValidationHandlers {
     private static final List<HttpValidationHandler> configurations = ServiceLoaderUtils.load(HttpValidationHandler.class);
 
+    public static void register(HttpValidationHandler handler) {
```

Comment:
Most of the times I call these methods simply `add`. But I may have slipped a few ones like `register`. Should we rename to `add`?